src/pybind/mgr/cephadm/module.py

```diff
@@ -672,6 +683,8 @@ def _do_upgrade(self):
                         self._save_upgrade_state()
                         return None
 
+                self._update_upgrade_progress(done / len(daemons))
```

Comment:
add this import to make the division a float type in py2?
```
from __future__ import division
```

Reply:
mgr is py3 only